web/src/components/Icons.js

```diff
@@ -11,14 +11,18 @@ import {
   faCog,
   faEdit,
   faExclamationTriangle,
-  faHelpSolid,
+  faQuestionCircle as faHelpSolid,
   faPlusCircle,
   faPrint,
   faSpinner,
-  faSignOut
+  faSignOutAlt as faSignOut
```

Comment:
More whoops on my part - there aren't `faHelpSolid`, `faSignOut`, or `faHelp` components in these libraries.  We were importing and renaming them previously, but I didn't notice that when I switched to the newer version and import format.